### DISCUSSION TITLE
CUDA 관련 문의

### DISCUSSION THREAD

**Discussion by FIVEYOUNGWOO** [Management001/How_to_install_CUDA-#3] [Q&A] [answered]
Discussion 열어서 죄송합니다.. 혹시 CUDA 관련해서 문의 드리고 싶은데 메일 알려주실 수 있으실까요?

**Comment by Management001** [comment, marked as answer]
아무렴요! kjt@chosun.ac.kr로 문의 주시면 감사하겠습니다.

**Reply by FIVEYOUNGWOO** [reply]
해당 메일로 따로 문의 드리겠습니다. 시간 내어주셔서 감사합니다 !!!
